docs/modules/ROOT/pages/connectors/camel-file-kafka-sink-connector.adoc

```diff
@@ -67,10 +67,17 @@ The camel-file sink connector has no converters out of the box.
 
 
 
-The camel-file sink connector has no transforms out of the box.
+The camel-file sink connector supports 0 transforms out of the box, which are listed below.
```

Comment:
Looks like something is wrong in the doc generator plugin. Shouldn't it be "1 transform" here?

Reply:
I need to double check

Reply:
Thanks for spotting